.vscode/settings.json

```
@@ -0,0 +1,4 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "typescript.enablePromptUseWorkspaceTsdk": true
+}
```

Moja propozycja:

```suggestion
{
  "typescript.tsdk": "node_modules/typescript/lib",
  "typescript.enablePromptUseWorkspaceTsdk": true,
  "editor.formatOnSave": true,
  "editor.defaultFormatter": "esbenp.prettier-vscode",
  "eslint.run": "onSave",
  "editor.codeActionsOnSave": {
    "source.fixAll": true
  }
}
```

bo i tak dodamy prettiera.